hugegraph-hbase/src/main/java/com/baidu/hugegraph/backend/store/hbase/HbaseSessions.java

```diff
@@ -809,7 +807,7 @@ private void dump(String table, Scan scan) throws IOException {
                     byte[] key = CellUtil.cloneQualifier(cell);
                     byte[] val = CellUtil.cloneValue(cell);
                     LOG.info("  {}={}", StringEncoding.format(key),
-                                        StringEncoding.format(val));
+                             StringEncoding.format(val));
```

Comment:
also wrap line before `StringEncoding.format(key)` for logic symmetry

Reply:
#1996